plugin/trino-bigquery/pom.xml

```diff
@@ -537,6 +537,7 @@
                                 <exclude>**/TestBigQueryCaseInsensitiveMapping.java</exclude>
                                 <exclude>**/TestBigQuery*FailureRecoveryTest.java</exclude>
                                 <exclude>**/TestBigQueryWithProxyTest.java</exclude>
+                                <exclude>**/TestBigQueryParentProjectId.java</exclude>
```

Comment:
*question: * is this test just temporary excluded (until we have infra in place)?

Reply:
Forgot to include in cloud-1 profile. Fixed.